### DISCUSSION TITLE
Pick vertex with lasso tool like in Blender

### DISCUSSION THREAD

**Discussion by noralis-bit** [pyvista/pyvista#4773] [Q&A]
Is it possible to batch select mesh vertices using the lasso tool, similar to what Blender does?
Because I want to manually add labels to the points in different parts of the mesh face. Or is there any other software or way to achieve this?

**Comment by banesullivan** [comment]
We currently do not have a lasso tool but only a rectangle select tool: 

- https://docs.pyvista.org/version/stable/api/plotting/_autosummary/pyvista.Plotter.enable_rectangle_visible_picking.html
- https://docs.pyvista.org/version/stable/api/plotting/_autosummary/pyvista.Plotter.enable_rectangle_through_picking.html
